### ISSUE TITLE
Github vs. Codeberg

### ISSUE THREAD

**Issue by mondlicht-und-sterne** [noahjutz/Splitfit#36]
me and [fossdd](https://github.com/fossdd) see your github account for splitfit [here](https://github.com/noahjutz/Splitfit). but we also see an account at [codeberg](https://codeberg.org/noahjutz/Splitfit) -> is it out of date ? that would be sad.

**Comment by fossdd** [comment]
Yeah, are you maybe able to update that, so that it's another source code base where people can submit issues and pull requests.

**Comment by noahjutz** [comment]
Yes, that repository is out of date. If there's a way to make it automatically mirror github with codeberg, I would implement that. I just don't want to manually update it again and again.

**Comment by mondlicht-und-sterne** [comment]
and why did you leave codeberg? what are the reasons?

Give your free project a free home. and. No tracking. Your data is not for sale. thats codeberg.

**Comment by noahjutz** [comment]
I'll consider switching to codeberg, but I can't promise anything. The main thing keeping me on github at the moment is the large community, among other things.

**Comment by mondlicht-und-sterne** [comment]
they are also available on codeberg

**Comment by noahjutz** [comment]
I'll think about it. Thank you for bringing my attention to this issue.

**Comment by noahjutz** [comment]
Saving this for later: [Mirroring github-codeberg](https://gist.github.com/rvl/c3f156e117e22a25f242)

Whether I switch to codeberg or not, I'd like to push to both remotes.
